customTests/custom-test-setup.sh

```diff
@@ -95,10 +95,22 @@ else
 			wget https://www.cs.virginia.edu/stream/FTP/Code/stream.c
 		fi
 
+		SKU=$( curl -H Metadata:true --max-time 10 -s "http://169.254.169.254/metadata/instance/compute/vmSize?api-version=2021-01-01&format=text")
+		if echo $SKU | grep "HX176rs"; then
+			BUILD=ZEN4
+		elif echo $SKU | grep "HB120rs_v3"; then
+			BUILD=ZEN3
+		elif echo $SKU | grep "HB120rs_v2"; then
```

Comment:
how do we handle the smaller VM sizes?

Reply:
Not handled in this iteration but when we get to that item we will have to expand this

Reply:
Should we handle the full size HBv4 VM size since it will use the same thread locations?

Reply:
Yea I just committed this